flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/api/stream/sql/JoinTest.scala

```diff
@@ -573,8 +575,8 @@ class JoinTest extends TableTestBase {
             term("select", "a", "b", "proctime")
           ),
           term("where",
-            "AND(=(a, a0), >=(PROCTIME(proctime), -(PROCTIME(proctime0), 3600000)), " +
-              "<=(PROCTIME(proctime), +(PROCTIME(proctime0), 3600000)))"),
+            "AND(=(a, a0), >=(PROCTIME(proctime), -(PROCTIME(proctime0), 3600000:INTERVAL HOUR))," +
+              " <=(PROCTIME(proctime), +(PROCTIME(proctime0), 3600000:INTERVAL HOUR)))"),
           term("join", "a, proctime, a0, b, proctime0"),
```

Comment:
nit: whitespace

Reply:
the blank was moved from end to front, because the line exceeds limit.